### PR TITLE
Profile screen navigation

### PR DESCRIPTION
**Pull request by nate-thegrate**
I made a couple tweaks to the screen navigation; everything should be working now!

### CODE REVIEW

File: lib/views/admin_portal/admin_portal.dart
Comment (nate-thegrate) [inline lib/views/admin_portal/admin_portal.dart:18]: I added a button to the Participant and Admin home screens that links to the profile page:
profile button

We can access the profile screen using `navigator.push()`, and Flutter will automatically add a "go back" button to the AppBar.

**Comment by mbeargie1238** [comment]
LGTM!

**Review by mbeargie1238 — APPROVED** [review]
LGTM!

**Comment by nate-thegrate** [comment]
> LGTM!

Sweet, thanks @mbeargie1238! I'll let Hongbo merge this into his branch when he's ready 👍

**Pull request merged**
